app/models/user.rb

```diff
@@ -35,4 +35,12 @@ def random_card
       user.cards.random_for_review.first
     end
   end
+
+  def self.notify_pending_cards
+    scope :condition, -> { where('cards.review_date <= ?', Time.zone.now) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.